python/AppTop/AppTop.py

```diff
@@ -95,9 +87,9 @@ def Init():
             jesdTxDevices = self.find(typ=jesd.JesdTx)
             dacDevices    = self.find(typ=ti.Dac38J84)
             lmkDevices    = self.find(typ=ti.Lmk04828)
-            appCore       = self.find(typ=appCommon.AppCore)
+            appCore       = self.find(typ=AppCore.AppCore)
```

Comment:
Because you have `import AppTop    as appTop`, I think this should be `appCore       = self.find(typ=appTop.AppCore)`

Reply:
Done.